server/index.js

```diff
@@ -70,6 +71,22 @@ const createApp = () => {
 
   // static file-serving middleware
   app.use(express.static(path.join(__dirname, '..', 'public')))
+  app.use(require('body-parser').text())
```

Comment:
can we move this up with the body parsing middleware. they have the comment   "// body parsing middleware" above them

Reply:
done & done